README.md

```diff
@@ -0,0 +1,106 @@
+# @tools-aoeur/eslint-config
+
+## Goals
+
+1. Find errors that are detectable with static analysis.
+2. Make reading code easier by providing consistent code style.
+3. Make writing code faster by leveraging auto fix wherever possible.
+
+## Getting started
+
+1. `npm install eslint @tools-aoeur/eslint-config --save-dev`
+2. Setup your project config in `eslint.config.mjs`:
+
+```js
+import { getPresets } from '@tools-aoeur/eslint-config';
+
+export default [
+  ...(await getPresets(
+    // Base config
+    'typescript', // or 'javascript'
+
+    // Optional extensions
+    'lodash',
+    'react',
+    'cssModules',
+    'jest',
+    'cypress',
+    'vitest'
+  )),
```

Comment:
For the simplest cases, this can be reduced even further:

```tsx
import {getPresets} from '@tools-aoeur/eslint-config';

export default await getPresets('typescript', 'react');
```